lib/ansible/modules/clustering/k8s/k8s.py

```diff
@@ -116,7 +116,9 @@
     state: present
     src: /testing/deployment.yml
 
-- name: Read definition file from the Ansible controller file system
+- name: >-
+    Read definition file from the Ansible controller file system
+    or by reading the definition from a file encrypted with Ansible Vault.
```

Comment:
I would make this a separate sentence that notes it will be handled properly if it's vault encrypted, ie

```
Read definition file from the Ansible controller file system. 
If the definition file has been encrypted with Ansible Vault it will automatically be decrypted.
```